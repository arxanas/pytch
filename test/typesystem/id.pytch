let id(x) = x
show_type(id)
