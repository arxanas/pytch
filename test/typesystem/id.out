line 2: FunctionTy(reason=TodoReason(todo='infer_lambda'), domain=PVector([ExistentialTyVar(reason=TodoReason(todo='parameter ty'), name='param_0')]), codomain=ExistentialTyVar(reason=TodoReason(todo='return ty'), name='return'))
